src/app/service-providers/service-providers.component.html

```diff
@@ -32,7 +34,7 @@ <h5>{{serviceProvider.name}}</h5>
               <button *ngIf="isAdmin()" class="left genric-btn primary" (click) = "deleteService(serviceProvider.id)" style="margin-left: 10px; background-color: darkred;">Delete</button>
               <a [routerLink]="['/service-providers',serviceProvider.id]" class="right genric-btn primary">Read more</a>
               <a *ngIf="isCustomer()" [routerLink]="['/create-contract', serviceProvider.id]" style="margin-right: 10px; background-color: #4faadd;" class="right genric-btn primary">Create contract</a>
-              <button *ngIf="isAdmin() || isCurrentProvider(serviceProvider.userDTO.id)" class="right genric-btn primary" [routerLink] = "['/update-service-provider',serviceProvider.id]" style="margin-right: 10px; background-color: darkgreen">Update</button>
+              <button *ngIf="isAdmin() || isCurentProvider(serviceProvider.userDTO.id)" class="right genric-btn primary" [routerLink] = "['/update-service-provider',serviceProvider.id]" style="margin-right: 10px; background-color: darkgreen">Update</button>
```

Comment:
isCurentProvider   Current?

Reply:
I named method wrong)
I`ll fix it in a few minutes